.travis.yml

```diff
@@ -13,6 +16,7 @@ env:
   - EMBER_TRY_SCENARIO=ember-1.13
   - EMBER_TRY_SCENARIO=ember-lts-2.4
   - EMBER_TRY_SCENARIO=ember-lts-2.8
+  - EMBER_TRY_SCENARIO=ember-lts-2.12
```

Comment:
This scenario is not defined yet in https://github.com/kaliber5/ember-responsive-image/blob/master/config/ember-try.js

Reply:
My bad, I'll add this scenario to ember-try.